apps/email/js/cards/compose.js

```diff
@@ -749,6 +742,37 @@ return [
       else {
         this.attachmentsContainer.classList.add('collapsed');
       }
+      this.updateAttachmentsAriaLabel();
+    },
+
+    /**
+     * Update the label used for accessibility that describes the attachments
+     * included.
+     */
+    updateAttachmentsAriaLabel: function() {
+      if (this.composer.attachments.length <= 1) {
+        mozL10n.setAttributes(this.attachmentsContainer,
+          'compose-attachments-container');
+        return;
+      }
+      var kilobytes = Math.ceil(this.calculateTotalAttachmentsSize() / 1024);
+      mozL10n.setAttributes(this.attachmentsContainer,
+        'compose-attachments-container-kib', { kilobytes: kilobytes });
+    },
+
+    /**
+     * Calculate the total size of all attachments included.
+     */
+    calculateTotalAttachmentsSize: function() {
+      var totalSize = 0;
+      if (!this.composer.attachments) {
```

Comment:
I know this is just movement of existing code, but let's take this opportunity to clean it up and remove this if test. attachments is always there, just zero length, and other code in this file just do length checks anyway without checking existence. I also confirmed on an empty compose, composer.attachments exists and is a zero length array.